lib/prepare.js

```diff
@@ -33,23 +33,18 @@ export async function prepare(version, packageVsix, logger, cwd) {
 
     logger.log(`Packaging version ${version} to ${packagePath}`);
 
-    const options = [
-      'package',
+    /** @type {import('@vscode/vsce/out/package').IPackageOptions} */
+    const options = {
+      cwd,
       version,
-      '--no-git-tag-version',
-      '--out',
-      packagePath,
-    ];
+      out: packagePath,
+    };
     if (isTargetEnabled()) {
-      options.push('--target', process.env.VSCE_TARGET);
+      // @ts-ignore: target is supported in IPackageOptions
```

Comment:
```suggestion
      // @ts-expect-error: target is supported in IPackageOptions
```

So that I'll be "notified" if it's ever fixed.